Classes/Service/ProcessingService.php

```diff
@@ -269,8 +269,8 @@ public function getNodesFromListWithTree(string $listOfNodes, array $parentPoint
     protected function getParentPointerAsString(array $parentPointer): string
     {
         if (isset($parentPointer['sheet'])) {
-            $flexformPointerString =
-                $parentPointer['table'] . ':' .
+            $flexformPointerString
+                = $parentPointer['table'] . ':' .
```

Comment:
I'd like to have the dot on first in new line.
```
    = $parentPointer['table'] . ':'
    . $parentPointer['uid'] . ':'
    . $parentPointer['sheet'] . ':'
    . $parentPointer['sLang'] . ':'
```

Reply:
resolved via sprint